grpclb/src/test/java/io/grpc/grpclb/GrpclbLoadBalancerTest.java

```diff
@@ -601,15 +601,20 @@ public void grpclbWorking() {
     assertSame(error1, picker6.result.getStatus());
 
     // Update backends, with a drop entry
-    List<InetSocketAddress> backends2 = Arrays.asList(
-        new InetSocketAddress("127.0.0.1", 2030), null);
+    List<InetSocketAddress> backends2 =
```

Comment:
This test code may be too long. Maybe it's testing multiple things that could be separated?